dash/dash.py

```diff
@@ -589,19 +601,22 @@ def interpolate_index(self, **kwargs):
                             {app_entry}
                             {config}
                             {scripts}
+                            {renderer}
                             <div id="custom-footer">My custom footer</div>
                         </body>
                     </html>
                     '''.format(
                         app_entry=kwargs.get('app_entry'),
                         config=kwargs.get('config'),
-                        scripts=kwargs.get('scripts'))
+                        scripts=kwargs.get('scripts'),
+                        renderer=kwargs.get('renderer'))
 
         :param metas: Collected & formatted meta tags.
         :param title: The title of the app.
         :param css: Collected & formatted css dependencies as <link> tags.
         :param config: Configs needed by dash-renderer.
         :param scripts: Collected & formatted scripts tags.
+        :param renderer: The DashRenderer instance.
```

Comment:
For clarity, perhaps:
`:param renderer: A script tag that instantiates the DashRenderer.`